infra/loadtests/2hour-prod.js

```diff
@@ -0,0 +1,63 @@
+import http from 'k6/http';
+import { group } from 'k6';
+
+export const options = {
+  stages: [
+    { duration: '10m', target: 1},  // 1 virtual users
+    { duration: '10m', target: 2},
+    { duration: '10m', target: 1},
+    { duration: '10m', target: 3},
+    { duration: '10m', target: 1},
+    { duration: '10m', target: 4},
+    { duration: '10m', target: 1},
+    { duration: '10m', target: 5},
+    { duration: '10m', target: 1},
+    { duration: '10m', target: 6},
+  ],
+  thresholds: {
+    http_req_failed: ['rate<0.01'], // http errors should be less than 1%
+    http_req_duration: ['p(95)<1500'], // 95 percent of response times must be below 1500ms
+  },
+};
+
+export default function () {
+  group("testnet EVM", function() {
+    let testnetEvmContract = {"contractAddress":"0x83458E8Bf8206131Fe5c05127007FA164c0948A2"} // testnet contract EVM
+    
+    let response_evm_testnet = http.post('https://contract-pinger-mainnet-1094058868047.europe-west1.run.app/evm_no_check', JSON.stringify(testnetEvmContract), {
+      headers: { 'Content-Type': 'application/json' },
+    }) 
+
+    console.log("EVM Testnet: " + response_evm_testnet)
+  });
+
+  group("testnet Solana", function() {
+    let testnetSolContract = {"contractAddress":"<add_sol_contract>"} // testnet contract SOL
+    
+    let response_sol_testnet = http.post('https://contract-pinger-mainnet-1094058868047.europe-west1.run.app/solana_no_check', JSON.stringify(testnetSolContract), {
+      headers: { 'Content-Type': 'application/json' },
+    })
+
+    console.log("SOL Testnet: " + response_sol_testnet)
+  });
+
+  group("mainnet EVM", function() {
```

Comment:
Let's skip mainnet load tests for now, each call is 3+ USD :) Even on Solana that would be expensive.